tests/test_dockerfile.py

```diff
@@ -0,0 +1,392 @@
+import sys
+import os
+import pytest
+from entrypoint_hook import CommandNotFound
+
+EXECUTABLES_FOLDER = "/usr/local/bin/"
+
+def abs_path(executable):
+    """Build manually (expected) executable absolute path"""
+    return os.path.join(EXECUTABLES_FOLDER, executable)
+
+def test_entrypoint_executables(hook):
+    """
+    Basic test without configuration to check
+    if entrypoint run each dogecoin executables.
+    """
+    ## Constant variable for test
+    test_environ = {
+        "DATADIR" : "/dogecoin/.dogecoin",
+        "USER" : "dogecoin",
+        "PATH" : os.environ["PATH"],
```

Comment:
not sure about `os.environ` - can we do this with a fixture instead?

Reply:
Same problem. If shared with the host, yes it may be an issue, if it's shared by the container it shouldn't be a concern.

Reply:
Let me ask it differently: what benefit does this give over a static fixture? Do we need this to be a working path? If yes, for what?

Reply:
The entrypoint use `PATH` to resolve executable location in `shutil.which`:
https://github.com/dogecoin/docker/blob/059cf4a86b60644225339bcd7a27930687837316/1.14.5/bullseye/entrypoint.py#L25

Entrypoint need at least `/usr/local/bin` in `PATH` to work properly [in a container].